DependencyInjection/Configuration.php

```diff
@@ -131,12 +161,13 @@ private function createPluginsNode()
      */
     private function createBindNode()
     {
-        return $this->createNode('bind')
-            ->useAttributeAsKey('name')
-                ->prototype('array')
+        return $this
+            ->createNode('bind')
                 ->useAttributeAsKey('name')
-                ->prototype('variable')->end()
-            ->end();
+                ->prototype('array')
+                    ->useAttributeAsKey('name')
+                    ->prototype('variable')->end()
+                ->end();
```

Comment:
also, the double prototype seems not great. Can you give an example of what you want to support?